src/umpire/util/AllocationMap.cpp

```diff
@@ -87,25 +87,34 @@ AllocationMap::remove(void* ptr)
 AllocationRecord*
 AllocationMap::findRecord(void* ptr)
 {
-  auto record = m_records.atOrBefore(reinterpret_cast<uintptr_t>(ptr));
 
-  if (record.value) {
-    void* parent_ptr = reinterpret_cast<void*>(record.key);
-    auto alloc_record =
-      reinterpret_cast<Entry>(record.value->back());
+  AllocationRecord* alloc_record;
```

Comment:
should initialize to nullptr? seems that previous behavior was to return nullptr if not the record was not found